packages/config/__tests__/ProfileInfo.TeamConfig.test.ts

```diff
@@ -277,7 +278,7 @@ describe("TeamConfig ProfileInfo tests", () => {
             const profInfo = createNewProfInfo(teamProjDir);
             await profInfo.readProfilesFromDisk();
             const desiredProfType = "tso";
-            const profAttrs = profInfo.getDefaultProfile(desiredProfType);
+            const profAttrs = profInfo.getDefaultProfile(desiredProfType) as IProfAttrs;
```

Comment:
Just to clarify, (and we probably have discussed this already on the CLI standup) 😢 
This means that Zowe Explorer will need to do something similar when calling the `getDefaultProfile` since we are changing the function signature.

I'm afraid I can't recall were we landed on whether this is an impactful change or not 😋

Reply:
After some discussion, it seems that this could be treated as a big fix since we were already returning `null` from the `getDefaultProfile` method.

Q: Wasn't there some other method in which we could make the `profile` property optional?

Reply:
that's a ZE-only thing